M/MaximumIndependentSet/build_tarballs.jl

```diff
@@ -0,0 +1,36 @@
+using BinaryBuilder
+
+name = "MaximumIndependentSet"
+version = v"0.1"
+sources = [
+    GitSource("https://github.com/claud10cv/MaximumIndependentSet.git", "3b37c113bd2e1b93bf38808f036632a731f0349b"),
+]
+
+script = raw"""
+cd ${WORKSPACE}/srcdir/MaximumIndependentSet
+make -j${nproc}
+make install
+"""
+
+platforms = supported_platforms()
+filter!(t -> os(t) in ["linux"], platforms)
+filter!(t -> arch(t) == "x86_64", platforms)
+filter!(t -> isnothing(libc(t)) ||  libc(t) == "glibc", platforms)
```

Comment:
This is a slightly awkward way of just specifying the one platform, but it works. Out of curiosity though, is there a reason to not build on the other platforms?

Reply:
The makefile is a hard-coded, hand-written one. It only works on linux for now. I will work on creating a cmake script to enable cross-compile later this week

Reply:
I am testing a cmake build and will see if I can get to coompile on more platforms